config/initializers/hyrax.rb

```diff
@@ -197,7 +197,7 @@
   # config.audit_user_key = 'audituser@example.com'
   #
   # The banner image. Should be 5000px wide by 1000px tall
-  # config.banner_image = 'https://cloud.githubusercontent.com/assets/92044/18370978/88ecac20-75f6-11e6-8399-6536640ef695.jpg'
+  config.banner_image = 'images/homepage_banner.png'
```

Comment:
Can we check for a config variable here in case we want to switch it on the fly without pushing commits?  We don't even need to commit the corresponding key into the config YAML if we do a safe "dig" on the lookup, like:

`ESSI.config.dig(:essi, :homepage_banner) || 'images/homepage_banner.png'`

That way I could change the banner with a change to the secrets file and a restart.

Reply:
@randalldfloyd changes made, force-pushed